### PR TITLE
Add selected language filter

### PR DESCRIPTION
**Pull request by mlorenzano**
This solves #9

### CODE REVIEW

File: ElcoLinguist/src/mainwindow.cpp [hunk 1]
```diff
@@ -378,6 +392,24 @@ void MainWindow::searchContext(const QString &context, const QString &page)
     m_filterSearch->setFilter(context, page);
 }
 
+void MainWindow::setIndexFilteredLang()
+{
+    QVector<unsigned int> indexFilteredLang;
+    auto languages = m_languagesModel.languages();
```
Comment (AndreaRicchi) [inline ElcoLinguist/src/mainwindow.cpp]: `languages` should be defined as `const`

File: ElcoLinguist/include/tablefilter.h [hunk 2]
```diff
@@ -16,14 +16,17 @@ class TableFilter : public QSortFilterProxyModel
...
 protected:
     bool filterAcceptsRow(int source_row, const QModelIndex &source_parent) const override;
+    bool filterAcceptsColumn(int source_column, const QModelIndex &source_parent) const override;
 
 private:
     LanguagesModel *m_model;
     QVector<Language> m_languages;
     QString m_currentContext;
     QString m_currentPage;
+    QVector<unsigned int> m_indexFilteredLang;
```
Comment (guerinoni) [inline ElcoLinguist/include/tablefilter.h]: Can you use `uint16_t` insted of `unsigned int`

File: ElcoLinguist/include/tablefilter.cpp [hunk 3]
```diff
@@ -62,3 +67,11 @@ bool TableFilter::filterAcceptsRow(int source_row,
 
     return filterOk;
 }
+
+bool TableFilter::filterAcceptsColumn(int source_column,
+                                      const QModelIndex &source_parent) const
+{
+    Q_UNUSED(source_parent)
+
```
Comment (guerinoni) [inline ElcoLinguist/include/tablefilter.cpp]: empty line

**Comment by mlorenzano** [comment]
I have updated the code as suggested :)

**Pull request merged**